docs/ai/toc.yml

```diff
@@ -85,6 +85,8 @@ items:
     href: quickstarts/evaluate-ai-response.md
   - name: "Tutorial: Evaluate a response with response caching and reporting"
     href: tutorials/evaluate-with-reporting.md
+  - name: "Tutorial: Evaluate a response's safety"
```

Comment:
Would `Evaluate the content safety of a model's response` read better?

Reply:
And if yes, should we also change the title of the other tutorial above to something like `Evaluate the quality of a model's response (with response caching and reporting)`...

Basically, wondering if the titles should align and make it easier to know that we have 2 kinds of evaluators - quality and safety (just by looking at the toc tree under the main Evaluation topic)

However, the "with response caching and reporting" part is already different and we probably don't want to repeat it on both... So not sure what's best - perhaps I am overthinking it :)